cl/stats/management/commands/send_events_email.py

```diff
@@ -17,11 +17,21 @@ def handle(self, *args, **options) -> None:
         today = now()
         yesterday = today - timedelta(days=1)
         events = Event.objects.filter(date_created__gte=yesterday)
-        if events.count() > 0:
-            template = loader.get_template("emails/events_email.txt")
-            send_mail(
-                subject=f"CourtListener events email for {today.date().isoformat()}",
-                message=template.render({"events": events}),
-                from_email=settings.DEFAULT_FROM_EMAIL,
-                recipient_list=[settings.PARTNERSHIP_EMAIL_ADDRESS],
-            )
+        if not events.count():
+            return
+
+        # Filter out API-related user milestones since those are now handled by
+        # the Zoho integration. This keeps only global API events, global
+        # webhook events, and user tracking webhook milestones.
+        events_for_email = [
```

Comment:
I think it’d be helpful to have a test that confirms the correct behavior of this filter. That way, if we ever change the event description, we can catch that the filters also need to be updated.